app/src/main/java/com/example/memeitupapp/ui/gridmemes/model/GridMemesModel.kt

```diff
@@ -1,11 +1,11 @@
 package com.example.memeitupapp.ui.gridmemes.model
 
 import com.globant.domain.entity.Meme
-import com.example.memeitupapp.data.repository.MemeService
+import com.globant.data.service.MemeService
 import com.example.memeitupapp.ui.contract.GridMemesContract
 import com.globant.domain.util.Result
 
-class GridMemesModel(private val memesService: MemeService) : GridMemesContract.Model {
+class GridMemesModel(private val memesService: com.globant.data.service.MemeService) : GridMemesContract.Model {
```

Comment:
this is not necessary **com.globant.data.service.**

Reply:
Refactored!